### PR TITLE
Rewrite README

### LINKED ISSUE #23: Quickstart examples
**Problem**
Newcomers should be able to copy-paste single-script examples from the README to quickly try things out, as in the [tartiflette-aiohttp usage example](https://github.com/dailymotion/tartiflette-aiohttp#usage) usage example.

**Solution**
- Add a single-script example of a standalone `TartifletteApp` at the top of the README.
- Add a single-script example of a `TartifletteApp` mounted onto a `Starlette` app in a subsection.

### LINKED ISSUE #27: Official logo or banner
[`tartiflette-aiohttp`](https://github.com/dailymotion/tartiflette-aiohttp) has a banner displayed at the top of its README (see below). Having an equivalent one for `tartiflette-starlette` would be nice to enhance the attractiveness of the repo.

Related to #20 

@tsunammis Do you or someone else at Dailymotion still have the art files for the banner? I can make a `tartiflette-starlette` version. :-)

### PR DESCRIPTION
**Pull request by florimondmanca**
Fixes #23, fixes #27.

Tip: click "View file" to view the new README.

### CODE REVIEW

**Comment by tsunammis** [comment]
@florimondmanca I have only one question regarding the link between `tartiflette` and `tartiflette-starlette`. I would rather embed `tartiflette` as a dependency of `tartiflette-starlette`. `tartiflette-starlette` depends on a version of `tartiflette` regarding the Engine creation and the execution.

What do you think?

**Comment by florimondmanca** [comment]
Yep, so as discussed offline I'll add Tartiflette as a dependency and add notes on how to install its C dependencies too. I'll update the README to present `tartiflette-starlette` as a "wrapper" instead of an "adapter", so that people can simply go to `tartiflette-starlette` and get the ground running. :+1:

File: README.md [hunk 1]
```diff
@@ -1,137 +1,204 @@
...
+Response:
 
-```bash
-curl \
-  -X POST \
-  -H "Content-Type: application/json" \
-  -d '{"query": "{ hello(name: \"Chuck\") }"}' \
-  http://localhost:8000
+```json
+{ "data": { "hello": "Hello, Chuck!" }, "errors": null }
```
Comment (Maximilien-R) [inline README.md]: I think there is a little mistake here. In a GraphQL response, if no errors were encountered during the requested operation, the errors entry should not be present in the result (cf. [spec](https://graphql.github.io/graphql-spec/June2018/#sec-Errors)).
```suggestion
{ "data": { "hello": "Hello, Chuck!" } }
```
The GraphiQL image below should be updated to 🙏

Comment (florimondmanca) [reply README.md]: Good catch! I’ll change these. :-)

Comment (florimondmanca) [reply README.md]: Tracked in #28, if that's fine with you I suggest we leave this as is and I'll update it when #28 is solved (which will all happen before releasing 0.2).

Comment (florimondmanca) [reply README.md]: Just been fixed, actually. :+1:

**Pull request merged**